src/Sulu/Bundle/SecurityBundle/Tests/Unit/EventListener/SuluSecurityListenerTest.php

```diff
@@ -37,7 +38,7 @@ public function setUp()
     {
         parent::setUp();
 
-        $this->securityChecker = $this->prophesize('Sulu\Bundle\SecurityBundle\Permission\SecurityCheckerInterface');
+        $this->securityChecker = $this->prophesize(SecurityCheckerInterface::class);
```

Comment:
Just talked to @chirimoya, since we are supporting PHP 5.5 now we can use this ;) Makes a lot of stuff a lot easier :)